src/app/services/api.ts

```diff
@@ -131,14 +123,14 @@ export class ApiService {
     if (sortBy !== null) { queryString += `&sortBy=${sortBy}`; }
     if (secondarySort !== null) { queryString += `&sortBy=${secondarySort}`; }
     if (populate !== null) { queryString += `&populate=${populate}`; }
-    if (queryModifier !== {}) {
+    if (Object.keys(queryModifier).length !== 0) {
       Object.keys(queryModifier).forEach(key => {
```

Comment:
i wonder what would happen if you just got rid of the if conditional and let it iterate over Object.keys?

I mean, if there's no keys, it will stop iterating at the 0th key right?

not that it matters here, but I though it was interesting.

Reply:
I didn't know that it works that way, I thought it would throw an error. That's good to know though.